packages/tools/devtools/devtools-example/src/Container.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ModelContainerRuntimeFactory } from "@fluid-example/example-utils";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { IContainer } from "@fluidframework/container-definitions";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+
+import { CollaborativeDevtoolExampleAppObjects } from "./FluidObject";
+
+/**
+ * The CollaborativeDevtoolExampleAppModel is the model for our Fluid object.
+ * It contains the data we want to share with other clients.
+ */
+export interface ICollaborativeDevtoolExampleAppModel {
+	readonly collaborativeObj: CollaborativeDevtoolExampleAppObjects;
+	readonly container: IContainer;
+}
+
+class CollaborativeDevtoolExampleAppModel implements ICollaborativeDevtoolExampleAppModel {
+	public constructor(
+		public readonly collaborativeObj: CollaborativeDevtoolExampleAppObjects,
+		public readonly container: IContainer,
+	) {}
+}
+
+const collaborativeObjId = "collaborative-obj";
+
+/**
+ * The CollaborativeTextContainerRuntimeFactory is the container code for our Fluid object.
+ */
+export class CollaborativeDevtoolExampleObjectsContainerRuntimeFactory extends ModelContainerRuntimeFactory<ICollaborativeDevtoolExampleAppModel> {
```

Comment:
Since this is specific to our example app, and it's the only runtime factory, I think we can probably just call this "RuntimeFactory".

Reply:
Renamed